docker-compose.yml

```diff
@@ -2,12 +2,14 @@ version: '3.7'
 
 services: # the different images that will be running as containers
   web: # service name
-    build: . # build the image for the web service from the dockerfile in parent directory.
+    build:
+      context: . # build the image for the web service from the dockerfile in parent directory.
+      target: development-stage
     volumes:
       - .:/app # map data and files from parent directory in host to plio directory in docker container
     env_file: # file where env variables are stored. Used as best practice so as not to expose secret keys
       - .env # name of the env file
     image: plio-frontend # name of the image
     ports:
-      - "${APP_PORT}:80"
+      - "${APP_PORT}:${APP_PORT}"
```

Comment:
this was correct before right? as you were exposing port 80 in DockerFile?

Reply:
Docker-compose will only be used for local development. So we're not really using nginx for local workspace and instead relying on `npm run serve` command.

Reply:
Got it - but where are we supplying the APP_PORT to be used for running `npm run serve` within the container? Once that is done, then mapping it to the port outside makes sense to me.

Reply:
oh yes, that's right. let me check.

Reply:
Done. Please check now.